### DISCUSSION TITLE
can i consume the a DLL written on 32bit windows environment?

### DISCUSSION THREAD

**Discussion by dasaintgray** [halildurmus/win32#687] [Q&A] [answered]

**Comment by timsneath** [comment, marked as answer]
There's currently limited support for using this in 32-bit environments, but there are some caveats. In particular, there isn't a Dart compiler to x86, only an interpreter (that is, neither `dart compile exe` nor `flutter build windows` support 32-bit environments. Given the ever-decreasing usage of 32-bit Windows, I plan to deprecate 32-bit support with the v4 release of win32 and only support 64-bit from v5 onwards.
